src/tours.ts

```diff
@@ -0,0 +1,88 @@
+import { TourId, TourStep } from "./model";
+
+// If you complete a tour then we don't show it again.
```

Comment:
We need to move these strings out to the bundle but leaving here for now for easy hot reload while we iterate.